docs/en/stack/ml/get-started/ml-getting-started.asciidoc

```diff
@@ -1,6 +1,53 @@
 [[ml-getting-started]]
 == Getting started with {ml}
 
+{ml-cap} features analyze your data and generate models for its patterns of
+behavior. The type of analysis that you choose depends on the questions or
+problems you want to address and the type of data you have available.
+
+[discrete]
+[[get-started-unsupervised]]
+=== Unsupervised {ml}
+
+There two types of analysis that can deduce the patterns and relationships
+within your data without training or intervention: _{anomaly-detect}_ and
+_{oldetection}_.
+
+<<xpack-ml,{anomaly-detect-cap}>> requires time series data. It constructs a
+probability model and can run continuously to identify unusual events as they
+occur. The model evolves over time; you can use its insights to forecast future
+anomalous events.
+
+<<dfa-outlier-detection,{oldetection-cap}>> does not require time series data;
+it identifies unusual points in a data set by analyzing how close each data
+point is to others and the density of the cluster of points around it. It does
+not run continuously; it generates a copy of your data set where each data point
+is annotated with an outlier score.
```

Comment:
Would it be a good idea to add an explanatory sentence about what the outlier score indicates? Something like: ...is annotated with an {olscore} that indicates to what extent a data point is an outlier compared to the others.